goon/code/datums/browserOutput.dm

```diff
@@ -149,7 +149,7 @@ var/list/chatResources = list(
 				var/list/row = connectionHistory[i]
 				if(!row || row.len < 3 || !(row["ckey"] && row["compid"] && row["ip"]))
 					return
-				if(world.IsBanned(row["ckey"], row["ip"], row["compid"], FALSE))
+				if(world.IsBanned(key=row["ckey"], address=row["ip"], computer_id=row["compid"], type=null, check_ipintel=FALSE))
```

Comment:
do keep in mind that this will always compile, but if the arguments for IsBanned change later, it'll runtime when the server is running.